httprouter/httprouter.go

```diff
@@ -234,7 +234,14 @@ func (r *HTTProuter) AddRawHTTPHandler(pattern, HTTPmethod string, handler http.
 func (r *HTTProuter) routerHandler(namespaceID string, accessType AuthAccessType,
 	handlerFunc RouterHandlerFn) func(w http.ResponseWriter, req *http.Request) {
 	return func(w http.ResponseWriter, req *http.Request) {
-		defer req.Body.Close()
+		var err error
+
+		defer func() {
+			if err := req.Body.Close(); err != nil {
+				http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
If we cannot close the body (because there is some error) I don't think we can write an error as reply. You might just ignore this error, no need to print it either since it might be the client disconnecting from the socket, and that's nothing we can solve.